OneCommunityGlobal/HighestGoodNetworkApp
# Sarath added TeamsOverview.css fixing the Teams UI (padding,cards)

# Description
The Teams page in admin/owner profile has the following UI issues - 
        - At 768x and below, the table is not centered. Too much left side margin and too little right side.
	- At 768px and below, that top 2 cards have no padding
	- At 575px and below, the table starts side scrolling. We need to reduce font size to prevent this.
	- AT 479px and below, the cards stack vertically. We can make them smaller so this doesn’t happen.
	- Around 400px and below, the members modal’s table cuts off and starts side scrolling.



## Related PRS (if any):
This frontend PR is related to the #develpment backend PR.

…

## Main changes explained:
- Added file TeamsOverview.css to 
- Added a TeamsOverview.css styling page to make the cards adjust to the UI instead of stacking vertically.
- Added a padding as required. The table goes to scrolling mode when the rest of the table is not visible in the viewport at lower resolutions.

…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. Click on other links and go to Teams page
6. Go to dev tools and verify the above changes in different resolutions, and also check the responsiveness of the page.


## Screenshots or videos of changes:


Before: 
https://www.loom.com/share/853a50e83a2f40a4a8c7cc065142fa0e

After Changes: 
https://www.loom.com/share/d7c3b8096dad460f9abfa70d5020e56b 
https://www.loom.com/share/f762567f1bf94c66b7b958ca7ae1456b
